build/patch-charts.sh

```diff
@@ -146,9 +146,14 @@ function patchGalley() {
   sed_wrap -i -e '/{{- if eq .Values.revision ""}}/,/{{- end }}/d' $deployment
 
   # multitenant
-  echo '  - apiGroups: ["maistra.io"]
+  echo '
+  # Maistra specific
+  - apiGroups: ["maistra.io"]
     resources: ["servicemeshmemberrolls"]
-    verbs: ["get", "list", "watch"]' ${HELM_DIR}/istio-control/istio-discovery/templates/clusterrole.yaml
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["route.openshift.io"]
+    resources: ["routes", "routes/custom-host"]
+    verbs: ["get", "list", "watch", "create", "delete"]' >> ${HELM_DIR}/istio-control/istio-discovery/templates/clusterrole.yaml
```

Comment:
do you need `update` too?

Reply:
@rcernich Strictly speaking, no. IOR doesn't update a route, it deletes and creates a new one. That said, I think it makes sense to add `update` too, for completeness.